src/super_gradients/training/models/detection_models/customizable_detector.py

```diff
@@ -172,9 +175,17 @@ def _get_pipeline(self, iou: Optional[float] = None, conf: Optional[float] = Non
 
         iou = iou or self._default_nms_iou
         conf = conf or self._default_nms_conf
+
+        # Ensure that the image size is divisible by 32.
+        if isinstance(self._image_processor, ComposeProcessing) and skip_image_resizing:
+            image_processor = self._image_processor.get_equivalent_compose_without_resizing()
+            image_processor.processings.append(DetectionAutoPadding(shape_multiple=(32, 32), pad_value=0))
```

Comment:
Two notes:
1) I think DetectionAutoPadding should be the first in the list of transforms (BC there could be some image permute operations at the end)
2) DetectionAutoPadding should be added inside `get_equivalent_compose_without_resizing`

Reply:
done here [3874c86](https://github.com/Deci-AI/super-gradients/pull/1605/commits/3874c8672d1af4b11403307c69d367348ccad499)